db.go

```diff
@@ -2604,9 +2605,23 @@ type LSMKeyStatistics struct {
 func (d *DB) ScanStatistics(ctx context.Context, lower, upper []byte) (LSMKeyStatistics, error) {
 	stats := LSMKeyStatistics{}
 	var prevKey InternalKey
+	const RATE = 50
+	const BURST = 1000
```

Comment:
Need to figure out what actual values to have here (I just put placeholders for now)